orion/pkg/domains/user_valid_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func TestValidFirstName(t *testing.T) {
 	// Checks for valid first names
-	user := domains.User{FirstName: "John", LastName: "Smith", Email: "gmail@gmail.com", Phone: "555-555-0100", IsGuardian: true}
+	user := domains.User{FirstName: "John", LastName: "Smith", Email: "gmail@gmail.com", Phone: "555-555-0100", IsGuardian: true, GuardianId: domains.NewNullUint(0)}
```

Comment:
nit: I think this line is too long.
Can you separate the lines into a block?
domain.User{
FirstName: ...,
...
}